### PR TITLE
fix: Dependency fix

### PR DESCRIPTION
**Pull request by jakirsshaikh**
Added Test and prod based dependency.
https://coupadev.atlassian.net/browse/CD-288917

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,8 +37,10 @@ def version():
     keywords='sand',
     install_requires=[
         'requests>=2.24.0',
-        'python-dateutil>=2.7.5'
-        'oauthlib>=3.2.0'
-        'requests-mock>=1.9.3'
+        'python-dateutil>=2.7.5',
+        'oauthlib>=3.2.0',
+        'requests-mock>=1.9.3',
```
Comment (johnwu96822) [inline setup.py:42]: Is this loaded in production? Is there a way to only load it in testing and not in production? In Ruby there is development dependencies, wonder if there's something like this in python.

**Comment by johnwu96822** [comment]
Please add the description and ticket to the PR.

File: setup.py (hunk 1 above)
Comment (jakirsshaikh) [reply setup.py]: added a way to separate dev/test dependency from required dependency.

**Review by apscolari — APPROVED** [review]
nothing to add, looks good. 
thank you
Ana

**Review by johnwu96822 — APPROVED** [review]

**Pull request merged**